tools/bulk_compression.py

```diff
@@ -0,0 +1,405 @@
+
+def deleteFile(fname):                                              # Delete a file in pwd
+    print(f"║ Deleting {fname}...{(66 - len(fname)) * ' '}║")
+    os.remove(f"./{fname}")
+
+
+# =================================================================================================
+
+options = {                                                         # Options listings
+    "1" : "Convert ISO to CSO",
+    "2" : "Convert ISO to CHD",
+    "3" : "Convert CUE/BIN to CHD",
+    "4" : "Convert CSO to CHD",
+    "5" : "Convert DVD CHD to CSO",
+    "6" : "Extract DVD CHD to ISO",
+    "7" : "Extract CD CHD to CUE/BIN",
+    "8" : "Extract CSO to ISO",
+    "9" : "Exit script",
+}
+
+sources = {                                                         # Source file extensions
+    "1" : "iso",
+    "2" : "iso",
+    "3" : "cue/bin",
+    "4" : "cso",
+    "5" : "chd",
+    "6" : "chd",
+    "7" : "chd",
+    "8" : "cso",
+}
+
+targets = {                                                         # Target file extensions
+    "1" : "cso",
+    "2" : "chd",
+    "3" : "chd",
+    "4" : "chd",
+    "5" : "cso",
+    "6" : "iso",
+    "7" : "cue/bin",
+    "8" : "iso",
+}
+
+reqs = {                                                            # Selection dependencies
+    "1" : ["maxcso"],
+    "2" : ["chdman"],
+    "3" : ["chdman"],
+    "4" : ["maxcso", "chdman"],
+    "5" : ["maxcso", "chdman"],
+    "6" : ["chdman"],
+    "7" : ["chdman"],
+    "8" : ["maxcso"],
+}
+
+# -------------------------------------------------------------------------------------------------
+
+print("╔===============================================================================╗")
+print("║       CSO/CHD/ISO/CUEBIN Conversion by Refraction and TheTechnician27         ║")
+print("║                           (Version Jul 16 2024)                               ║")
+print("╠===============================================================================╣")
+print("║                                                                               ║")
+print("║ PLEASE NOTE: This will affect all files in this folder!                       ║")
+print("║ Be sure to run this from the same directory as the files you wish to convert. ║")
+print("║                                                                               ║")
+
+for number, message in options.items():
+    print("║ ", number, " - ", message, f"{(70 - len(message)) * ' '}║")
+
+print("║                                                                               ║")
+print("╠===============================================================================╝")
+print("║                                                                                ")
+selection = input("║ Type the number corresponding to your selection then press ENTER: ")
+
+# -------------------------------------------------------------------------------------------------
+
+selection_num = 0
+try:
+    selection_num = int(selection)
+
+except:
```

Comment:
```suggestion
except ValueError: 
```
Codacy is complaining because you are capturing all exceptions, even keyboard interrupts and systemexit ones.
In this context will it actually cause a problem? Probably not, but less codacy warnings the better

Reply:
Done, and moved to new /bin/utils directory so that it ships with PCSX2. Completely missed that one; sorry.